utilities/transactions/pessimistic_transaction.cc

```diff
@@ -105,7 +105,12 @@ void PessimisticTransaction::Initialize(const TransactionOptions& txn_options) {
   commit_timestamp_ = kMaxTxnTimestamp;
 
   if (txn_options.commit_bypass_memtable) {
-    commit_bypass_memtable_threshold_ = 0;
+    // No need to optimize for empty transction
+    commit_bypass_memtable_threshold_ = 1;
+  } else if (txn_options.large_txn_commit_optimize_threshold !=
```

Comment:
This means for `large_txn_commit_optimize_threshold` to take effect, `commit_bypass_memtable` should be set to false, right?

This is not directly related to this PR, but I wonder if it makes sense to unify the boolean flag and the int flag into one int flag to simplify the configuration.